lib/ramble/ramble/util/editor.py

```diff
@@ -128,6 +128,8 @@ def try_env_var(var):
     if try_exec(exe, [exe] + list(args)):
         return
 
+    tty.die('  Here ')
```

Comment:
What's the purpose of this? Looks like maybe a left over debugging giving the next section raises an error?